### DISCUSSION TITLE
Fedora support?

### DISCUSSION THREAD

**Discussion by mihail727** [realmazharhussain/gdm-settings#18] [Q&A] [answered]
will there be support for fedora and when?

**Comment by realmazharhussain** [comment, marked as answer]
I do not think that there will be an rpm package. I might change my mind in the future but current plan is to have Flatpak and AppImage support. These formats support almost every distro.

Flatpak support is being discussed in issue #8 and AppImage support in issue #9.

**Edit:** A flatpak has been built. See [comment #1](https://github.com/realmazharhussain/gdm-settings/issues/8#issuecomment-1056412315) of issue #8.
**Edit 2:** The application is available on Flathub. The Flathub page for this application is [here](flathub.org/apps/details/io.github.realmazharhussain.GdmSettings).
